content/blog/2025/251028-stac-sprint-eu-recap.md

```diff
@@ -0,0 +1,77 @@
+
+- **[Datacube Extension Overhaul](https://github.com/stac-extensions/datacube/pull/32)**: A proposal to merge variables metadata in bands object to better align with how Zarr actually structures data. This change will make the datacube extension more intuitive and powerful for multidimensional datasets. This PR will probably require further discussion before merging, as it represents a breaking change in the extension.
+
+- **[EOPF STAC Patterns](https://github.com/EOPF-Sample-Service/eopf-stac/pull/54)**: ESA-specific best practices for their Earth Observation Processing Framework (EOPF) Zarr products, ensuring that Sentinel data in Zarr format can be properly cataloged and discovered.
+
+- **[STAC in Zarr](https://github.com/radiantearth/community-sprints/blob/main/14102025-esrin-rome-italy/sprint-notes/STAC%20in%20Zarr.md)**: Explorations into embedding STAC metadata directly inside Zarr stores, opening up fascinating possibilities for self-describing data archives.
+
+{{< img src="images/251025-stac-sprint-at-work.jpg">}}
+*Participants collaborating during the sprint sessions*
+
+## Connecting with the Zarr Community
+
+On Friday, October 17th, STAC sprint participants were invited to join the Zarr Adopter Summit, which was happening the same week in Rome. This was a perfect opportunity to share our progress and learnings with the broader Zarr community. Several sprint participants presented the work accomplished during the week, including the proposal for a STAC convention. This cross-pollination between communities—with STAC folks learning more about Zarr developments and Zarr folks understanding STAC use cases—exemplified the collaborative spirit that makes open standards work so well.
+
+## Beyond STAC
+
+While the technical achievements were substantial, some of the most valuable moments came from informal conversations and community building. On Wednesday evening, Development Seed and Eartmover sponsored a happy hour at Casadante in Rome, bringing together participants from both the STAC sprint and the concurrent Zarr Summit. Seeing people from Earth observation, climate science, and cloud infrastructure all discussing the same data challenges was remarkable.
+
+
+{{< img src="images/251025-stac-sprint-aperitivo.png">}}
+*Aperitivo at Casadante with STAC and Zarr communities*
+
+On Monday, ESA graciously provided a tour of their multimedia center and long-term archive facilities, including what may be one of the first catalogs of Earth observation data ever created. These glimpses into the history and future of Earth observation provided valuable context for our work.
```

Comment:
technically it was Tuesday, but IDK if it really matters!

Reply:
I was really shifted that week! :-D